15/umbraco-cms/reference/notifications/sendingallowedchildrennotifications.md

```diff
@@ -4,82 +4,6 @@ description: Example of how to use a SendingAllowedChildren Notification
 
 # Sending Allowed Children Notification
 
-The `SendingAllowedChildrenNotification` enables you to manipulate the Document Types that will be shown in the create menu when adding new content in the backoffice.
+The `SendingAllowedChildrenNotification` is no longer available in Umbraco 15.
```

Comment:
@AndyButland should probably be updated here as well:
https://github.com/umbraco/UmbracoDocs/blob/main/14/umbraco-cms/reference/notifications/sendingallowedchildrennotifications.md

Reply:
Good point, thanks.  I've updated here too.